### ISSUE TITLE
Running forever

### ISSUE THREAD

**Issue by abhimanyu3-zz** [rishikksh20/LSTM-Time-Series-Analysis#3]
The below code is taking for ever to run

for i in range(100):
    model.fit(trainX, trainY, nb_epoch=1, batch_size=batch_size, verbose=0, shuffle=False)
    model.reset_states()

Any fix for that.
